test/LendingPool.t.sol

```diff
@@ -1494,3 +1508,211 @@ contract VaultTest is LendingPoolTest {
         assertEq(amountLoaned, openPosition);
     }
 }
+
+/* //////////////////////////////////////////////////////////////
+                        GUARDIAN LOGIC
+////////////////////////////////////////////////////////////// */
+contract GuardianTest is LendingPoolTest {
+    using stdStorage for StdStorage;
+
+    address pauseGuardian = address(17);
+
+    function setUp() public override {
+        super.setUp();
+
+        // Set Guardian
+        vm.startPrank(creator);
+        pool.changeGuardian(pauseGuardian);
+        vm.stopPrank();
+
+        // Warp the block timestamp to 60days for smooth testing
+        vm.warp(60 days);
+    }
+
+    function testRevert_pause_depositInLendingPoolPaused() public {
```

Comment:
```suggestion
    function testRevert_ depositInLendingPool_Paused() public {
```
Naming convention for tests is always:
- testRevert_functionName_TestCase
- testSuccess_functionName_TestCase

Valid for all tests